airbyte-cdk/python/unit_tests/sources/streams/concurrent/test_concurrent_read_processor.py

```diff
@@ -560,6 +562,29 @@ def test_on_exception_return_trace_message_and_on_stream_complete_return_stream_
                 ),
             )
         ]
+        with pytest.raises(AirbyteTracedException):
+            handler.is_done()
+
+    def test_given_partition_completion_is_not_success_then_do_not_close_partition(self):
```

Comment:
I don't know why this is highlighted as something added as it is on master right now https://github.com/airbytehq/airbyte/blob/9f2d7172bcdcd2c4c366f45a9509ada93e50d26c/airbyte-cdk/python/unit_tests/sources/streams/concurrent/test_concurrent_read_processor.py#L564-L583